### PR TITLE
Tests for config/setup/log.go

### PR DESCRIPTION
**Pull request by Karthic-Hackintosh**
Tests for config/setup/log.go

### CODE REVIEW

File: config/setup/log_test.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+	if mid == nil {
+		t.Fatal("Expected middleware, was nil instead")
+	}
+
+	handler := mid(emptyNext)
+	myHandler, ok := handler.(caddylog.Logger)
+
+	if !ok {
+		t.Fatalf("Expected handler to be type Ext, got: %#v", handler)
```
Comment (mholt) [inline config/setup/log_test.go]: type Logger ;)

File: config/setup/log_test.go [hunk 2]
```diff
@@ -0,0 +1,134 @@
+package setup
+
+import (
+	"testing"
+
+	caddylog "github.com/mholt/caddy/middleware/log"
```
Comment (mholt) [inline config/setup/log_test.go]: Why the named import?

Comment (Karthic-Hackintosh) [inline config/setup/log_test.go]: The log.go had named import , just to be consistent i used it in the test too..

File: config/setup/log_test.go (hunk 1 above)
Comment (Karthic-Hackintosh) [inline config/setup/log_test.go]: Gosh !!! Ill make a commit with the correction

**Pull request merged**